README.md

```diff
@@ -44,6 +57,150 @@ In most cases this command must be run by root user or through `sudo` command.
 
 BSD systems are not tested well. If ALSA works, Oto should work.
 
+## Usage
+
+The two main components of Oto are a `Context` and `Players`. The context handles interactions with
+the OS and audio drivers, and as such there can only be **one** context in your program.
+
+From a context you can create any number of different players, where each player is given an `io.Reader` that
+it reads bytes representing sounds from and plays.
+
+> Note that a single `io.Reader` must **not** be used by multiple players.
```

Comment:
Is this a quote from somewhere?

Reply:
No but I like to use it sometimes to emphasis things.
Removed.